README.md

```diff
@@ -1,41 +1,66 @@
-## [Download TripleA](http://triplea-game.org/download/)
+# TripleA: Open Source Strategy Game Engine
```

Comment:
Perhaps "grand strategy" instead of just "strategy". The former is it's own category and consistent with the website IIRC

Reply:
Thanks @Cernelius . I think that is pretty compelling to just say strategy.  It is a bit niche I believe for a gamer to know the difference, therefore the distinction is potentially also unhelpful